projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -86,12 +94,16 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
     return ruleConditionComparators.filter((comparator) => Object.keys(comparator.labels).includes(this.selectedField.type));
   }
   get isComparatorHidden(): boolean {
-    return this.selectedField?.type === 'special';
+    return comparatorHiddenForConditionFieldType.includes(this.selectedField?.type);
   }
   get comparatorControl(): AbstractControl {
     return this.form.get('comparator');
   }
 
+  private get parameterControl(): AbstractControl {
```

Comment:
AbstractControl is generic type so could you please specify type for value of control using <>?

So I mean something like AbstractControl<type>

Reply:
done, specified the type of value